### ISSUE TITLE
Slot et al Panama leaf temperature response curve trait data

### ISSUE THREAD

**Issue by mdietze** [PecanProject/pecan#1273]
More trait data (potentially useful to @serbinsh and other NGEE Tropic folk), as well as a potential good example of grabbing data via DOI https://dx.doi.org/10.5479/data_stri/10088/30581  Martjin Slot has a dataset for Panama from his New Phyt paper on photosynthetic temperature response of 42 canopy species. The raw data contains almost 4000 photosynthesis measurements and contains stomatal conductance data as well.

**Comment by serbinsh** [comment]
Agreed...this was already on my list of files / data to add to BETYdb!

As you are no doubt aware now from Gitter activity, I am trying to sort out uploading larger datasets.  This would be a good one to have in BETY, in addition to the data we have been collecting in Panama which includes A-Ci, A-Q, and diurnal stomatal conductance, as well as SLA, N, leaf optics

-------------------------------------------------------------------------
Shawn P. Serbin
Associate Scientist
Environmental & Climate Sciences Department
Brookhaven National Laboratory
Upton, NY 11973-5000

Phone: +1-631-344-3165
Fax: +1-631-344-2060
Email: sserbin@bnl.gov<mailto:sserbin@bnl.gov>
Bio:  http://www.bnl.gov/envsci/bio/serbin-shawn.php
TEST Group: http://www.bnl.gov/TEST/
ORCiD: http://orcid.org/0000-0003-4136-8971
Twitter: @doctorjackpine, @test_bnl
-------------------------------------------------------------------------
________________________________
